### PR TITLE
Add option to freeze current 3d eye model

### PR DESCRIPTION
**Pull request by papr**
In some cases, one does not want the current eye model to change. This PR adds such an option to the 3d detector menu ("Freeze model") and the Pupil Detector Network API (#1390, `model_is_frozen` property).

**Warning**: Freezing the eye model will disable its ability to compensate for any kind of slippage.

You can freeze the eye model remotely by sending this notification to Pupil Capture:
```py
{
    "subject": "pupil_detector.set_property.3d",
    "name": "model_is_frozen",
    "value": True,  # set to False to unfreeze
    "target": "eye0",  # remove this line if you want to freeze the model both eyes
}
```

### CODE REVIEW

File: pupil_src/shared_modules/pupil_detectors/singleeyefitter/EyeModelFitter.cpp [hunk 1]
```diff
@@ -109,7 +117,8 @@ Detector3DResult EyeModelFitter::updateAndDetect(std::shared_ptr<Detector2DResul
     if (observation2D->confidence >= 0.7) {
 
         // allow each model to decide by themself if the new observation supports the model or not
-        auto circleAndFit = mActiveModelPtr->presentObservation(observation3DPtr, mAverageFramerate.getAverage()  );
+        // TODO: Pass freeze argument
```
Comment (romanroibu) [inline pupil_src/shared_modules/pupil_detectors/singleeyefitter/EyeModelFitter.cpp]: This `TODO` is obsolete, right?
```suggestion
```

**Comment by mattem86** [comment]
Hey, this does not work anymore, does it? It seems that there is no occurance of "model_is_frozen" anymore anywhere.

**Comment by papr** [comment]
It should still work but the API has changed with version 3.0 https://github.com/pupil-labs/pupil/releases/v3.0

**Comment by mattem86** [comment]
ah I see, the keyword also changed to "is_long_term_model_frozen" .. Thanks, I will test that!

**Pull request merged**